code/__HELPERS/unsorted.dm

```diff
@@ -480,6 +480,26 @@ Turf and target are separate in case you want to teleport some distance from a t
 			processing += A.contents
 			. += A
 
+//Gets all contents of contents but ignores those of atoms with the specified flag_1 (ex. atoms that protect their contents from explosions)
+/atom/proc/GetAllContentsIgnoreFlag(var/T, flag)
```

Comment:
just change `GetAllContents` to support checking a range of flags against flags_1 and flags_2 if it exists, dont' copypaste it